static/js/apps/explore/topics.json

```diff
@@ -296,23 +296,88 @@
       "examples": {
         "general": [
           {
-            "title": "How many people have degrees in Massachusetts?"
+            "title": "Educational Attainment in the US",
+            "url": "/insights/#p=country/USA&t=dc/topic/EducationalAttainment"
```

Comment:
We now share the URL between NL and insights.  Could you please strip out the `/insights/` and add `&q=` even for ones with p and t?  See other urls

Reply:
Done